show/plugins/cisco-8000.py

```diff
@@ -17,9 +17,17 @@
 except ImportError:
     PLATFORM_CLIS = []
 
+try:
+    from sonic_platform.cli import VENDOR_CLIS
+except ImportError:
+    VENDOR_CLIS = {}
+
 
 def register(cli):
     version_info = device_info.get_sonic_version_info()
     if version_info and version_info.get("asic_type") == "cisco-8000":
         for c in PLATFORM_CLIS:
             cli.commands["platform"].add_command(c)
```

Comment:
>platform [](http://example.com/codeflow?start=26&length=8)

platform is a special case of command. Please unified two for-block.

Reply:
The reason for the two separate loops is to not break backward compatibility in cisco platform specific code which only uses the PLATFORM_CLIS list.  New cisco platform specific code will use the VENDOR_CLIS dict, so until all cisco code has been migrated to using VENDOR_CLIS we really do need both for loops.

Reply:
Hi @qiluo-msft , if @gvlaemyn's explanation is acceptable could you help with the approval?

Reply:
Hi @qiluo-msft , if Geert's explanation is acceptable, please help with the approval.  Otherwise, we could arrange for a quick meeting to discuss.  Thank you.

Reply:
hi @qiluo-msft  and @kevinskwang , could you review and approve.

Reply:
What is the plan and ETA for "all cisco code has been migrated to using VENDOR_CLIS"? If I do not ask, will it happen finally?

Reply:
Reading the VENDOR_CLIS dictionary in the cisco-8000 plugin was added in this PR just for being able to add the "show techsupport npu" command.  My previous comment "until all cisco code has been migrated to using VENDOR_CLIS" may have given the impression that this PR is depending on a lot of internal cisco code to be modified: this is not the case.  I should have commented the PLATFORM_CLIS list may not be required to be read anymore in the cisco-8000 plugin at some future time, but currently we need both PLATFORM_CLIS and VENDOR_CLIS to be read.

Reply:
@qiluo-msft @kevinskwang Does the updated code in plugins/cisco-8000.py address your concerns? Thanks.